src/test/types.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const FILE_EXTENSION = ".linttest";
+
+export interface PositionInFile {
+   line: number;
+   col: number;
+}
+
+export interface LintError {
+   startPos: PositionInFile;
+   endPos: PositionInFile;
+   message: string;
+}
+
+export function errorComparator(err1: LintError, err2: LintError) {
```

Comment:
feels weird that this method implementation is in a file called `types.ts`. how about we name the file `lintError.ts` and move `FILE_EXTENSION` somewhere else